tailwind.config.ts

```diff
@@ -115,15 +118,36 @@ export default {
           from: { height: "var(--radix-collapsible-content-height)" },
           to: { height: "0" },
         },
+        aurora: {
+          from: {
+            backgroundPosition: "50% 50%, 50% 50%",
+          },
+          to: {
+            backgroundPosition: "350% 50%, 350% 50%",
+          },
+        },
       },
       animation: {
         "accordion-down": "accordion-down 0.2s ease-out",
         "accordion-up": "accordion-up 0.2s ease-out",
         "collapsible-down": "collapsible-down 0.2s ease-in-out",
         "collapsible-up": "collapsible-up 0.2s ease-in-out",
+        aurora: "aurora 60s linear infinite",
       },
     },
   },
 
-  plugins: [animate, setupInspiraUI],
+  plugins: [animate, setupInspiraUI, addVariablesForColors],
 } satisfies Config;
+
+// This plugin adds each Tailwind color as a global CSS variable, e.g. var(--gray-200).
+function addVariablesForColors({ addBase, theme }: any) {
```

Comment:
no need of adding this function. This is automatically added by inspira-ui plugins.

Reply:
No need to convert colours to CSS variables, you are duplicating stuff that Tailwind provides. 

Just use `theme`
```css
.element {
background: theme('colors.gray.400');
background: theme('colors.secondary.DEFAULT' / 50%);
}
```

Reply:
And this section is worst reading https://tailwindcss.com/docs/reusing-styles#extracting-classes-with-apply

Reply:
Hi @nathan-de-pachtere I'm aware of it already, but to differentiate huge chunk of css from the component bringing the chunk to style sction, just for more redability nothing else.

Reply:
My intial version is inline styles only, but I didn't like that big chuck in templete. That is why moved to Style section,

what is your suggestion on this @nathan-de-pachtere.

Reply:
I'm considering the experience for developers who’ll maintain this library in their projects. Adding another plugin to make variables work doesn’t seem ideal, which is why I suggest using only Tailwind, without addVariablesForColors(). Inspira-UI’s copy-paste model is for users to own their code without extra dependencies, so introducing another library goes against this core idea.

Separating structure (HTML) from design (CSS) keeps things clearer. Tailwind was made for inline classes in HTML, but for non-Tailwind projects, pure CSS would be cleaner and easier to maintain. Also, inline styles are outdated.

Your @apply it's just good, just saying about variables. I have no best suggestion for this big chunk of CSS.

Reply:
@nathan-de-pachtere Thanks for your reply, removed `addVariablesForColors` function as @rahul-vashishtha suggested, because @rahul-vashishtha  told it is pulled from the `@inspira-ui/plugins`. I checked the plugin as well it is too good.

Tahnks again @nathan-de-pachtere.

Reply:
Thanks you for your amazing work 😉 you are doing the hard 💪💪

Reply:
Thanks